build.sh

```diff
@@ -1,8 +1,17 @@
 #!/usr/bin/env bash
 
+CONFIGURATION=
+CI=0
+BUILD_NUMBER=
+RELEASE_LABEL=zlocal
+
 while true ; do
 	case "$1" in
-		-c|--clear-cache) CLEAR_CACHE=1 ; shift ;;
+		-c) CONFIGURATION=$2 ; shift ;;
```

Comment:
using `-c` for clearing the cache was pretty confusing, the Windows build.ps and every other repo uses -c for setting the Configuration so I did the same here